src/lib/src/grid3d/cellvol.cpp

```diff
@@ -3,50 +3,43 @@
 #include <pybind11/numpy.h>
 #include <pybind11/pybind11.h>
 
+#include <xtgeo/geometry.hpp>
+#include <xtgeo/grid3d.hpp>
 #include <xtgeo/xtgeo.h>
 
 namespace py = pybind11;
 
+namespace XTGeo::Grid3D {
+
 py::array_t<double>
-grid3d_cellvol(const size_t ncol,
-               const size_t nrow,
-               const size_t nlay,
-               const py::array_t<double> coordsv,
-               const py::array_t<float> zcornsv,
-               const py::array_t<int> actnumsv,
-               const int precision,
-               const bool asmasked = false)
+cellvol(const size_t ncol,
```

Comment:
Since this is now accessible as `XTGeo::Grid3D::cellvol` / `_internal.grid3d.cellvol` keeping the function name as `grid3d_cellvol` felt redundant